node/grpc/server_v2.go

```diff
@@ -101,6 +110,8 @@ func (s *ServerV2) StoreChunks(ctx context.Context, in *pb.StoreChunksRequest) (
 			return
 		}
 
+		s.metrics.ReportStoreChunksDataSize(size)
```

Comment:
what if the store operation gets reverted in L125?

Reply:
As a general rule of thumb, should we report incremental metrics if the operation as a whole fails? Or should we only report metrics for an operation if it is successful? (in another PR, you suggested that I should report latencies even when there are failures).

I can make this only report if the request ends up being valid, but I want to be consistent with the way we handle scenarios like this.

Reply:
As discussed offline, this will be left the way it currently is.